dlc-manager/src/contract/mod.rs

```diff
@@ -43,6 +43,8 @@ pub enum Contract {
     Closed(ClosedContract),
     /// A contract whose refund transaction was broadcast.
     Refunded(signed_contract::SignedContract),
+    /// A contract that was closed cooperatively.
+    CooperativeClose(CooperativeCloseContract),
```

Comment:
What is the point of that state? I feel the best would be to just not updated the state when proposing, and then update if/when the closing transaction is seen on-chain. Right now if you propose a cooperative close but it somehow doesn't end up being closed you will be in an incorrect state.